methods/secular/Secular_Requirement_Validation.ipynb

```diff
@@ -9,7 +9,8 @@
     "**Original code authored by:** David Bekaert, Heresh Fattahi, Eric Fielding, and Zhang Yunjun \n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    command = f'timeseries2velocity.py {set_cor_file} -o {cor_vel}'
</pre><p>This should be changed to the <code>iono_timeseries_file</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>